recipe/meta.yaml

```diff
@@ -22,8 +22,11 @@ requirements:
   host:
     - armadillo >=6.500.0
     - libblas
-    - libcblas
+    - libcblas  # [not win]
+    - libcblas ==3.8  # [win]
     - openmp
+    - vc ==14.*  # [win]
+    - vc ==9  # [win]
```

Comment:
As you can see here I added version of vc which are available. But getting error can you please explain.